### ISSUE TITLE
Won't Start As Is

### ISSUE THREAD

**Issue by kentroberts** [bradtraversy/angular60#4]
Needed to make a couple tweaks:
- specify type for function parameters hobby and i (perhaps due to newer version of TypeScript?)
- change reference for selenium-webdriver in package.json: 
      from "@types/selenium-webdriver": "^2.53.33"
          to  "@types/selenium-webdriver": "2.53.33"
      because of error:   node_modules/protractor/built/browser.d.ts(258,37): error TS2503: Cannot find namespace 'webdriver'.
         see: http://stackoverflow.com/questions/41232228/running-npm-start-fails-due-to-error

Thanks for a great tutorial Brad!

**Comment by I-am-Ironman** [comment]
@kentroberts and @bradtraversy sir the type of hobby and i are implicitly any ...so what do we define their types and where??

app/components/user.component.ts(45,12): error TS7006: Parameter 'hobby' implici
tly has an 'any' type.
app/components/user.component.ts(49,15): error TS7006: Parameter 'i' implicitly
has an 'any' type.

**Comment by kentroberts** [comment]
Yes, the type of any is implicit. Being new to Typescript, I assumed an "error" from the compiler indicates it is not allowed. Perhaps just a warning would be more appropriate.
Thanks for pointing that out.

**Comment by renzosalvador** [comment]
I did this: npm install @types/selenium-webdriver@2.53.33 and it worked.
